### PR TITLE
Design: Add settings page

### PR DESCRIPTION
**Pull request by 0xbaggi**
Done:
- Settings page created, with a custom widget/class settings_card.
- Implemented the onTapped() function on cards but there are no linked pages yet.

image

### CODE REVIEW

**Review by mikev-cw — APPROVED** [review]

File: lib/custom_widgets/settings_card.dart [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                            color: Colors.black,
+                            fontSize: 18.0,
+                            fontFamily: 'SF Pro Text',
+                            fontWeight: FontWeight.bold)),
+                    const SizedBox(height: 4),
+                    Text(settingDescription,
+                        style: const TextStyle(
+                          color: Colors.black,
+                          fontSize: 12.0,
+                          fontFamily: 'SF Pro Text',
```
Comment (FedericoBruzzone) [inline lib/custom_widgets/settings_card.dart:48]: I'd suggest you to use a macro for the font (and replace it where necessary, also in the other two widgets)

I apologize if you do not know what a macro is, I mean something like a global variable. 

**Let's talk about whether it is better to create a new file for the macros**

File: lib/pages/settings_page.dart [hunk 2]
```diff
@@ -11,9 +12,95 @@ class _SettingsPageState extends State<SettingsPage> {
...
+                        child: Row(
+                          children: const [
+                            SizedBox(width: 16),
+                            Icon(Icons.arrow_back_ios_new,
+                                size: 18.0, color: Colors.black),
+                            Text("Back",
+                                style: TextStyle(
+                                    color: Colors.black,
+                                    fontSize: 14.0,
+                                    fontFamily: 'SF Pro Text'))
```
Comment (FedericoBruzzone) [inline lib/pages/settings_page.dart:36]: Same thing above

File: lib/pages/settings_page.dart [hunk 3]
```diff
@@ -11,9 +12,95 @@ class _SettingsPageState extends State<SettingsPage> {
...
+                          ],
+                        ))),
+                const Expanded(
+                    child: Center(
+                  child: Text("Settings",
+                      style: TextStyle(
+                          color: Colors.black,
+                          fontSize: 16.0,
+                          fontFamily: 'SF Pro Text',
```
Comment (FedericoBruzzone) [inline lib/pages/settings_page.dart:45]: Same thing above

**Review by FedericoBruzzone — CHANGES_REQUESTED** [review]

**Comment by FedericoBruzzone** [comment]
Is the font size the same for all widgets?

**Comment by FedericoBruzzone** [comment]
Need rebase :D

**Pull request closed without merging**